README.md

```diff
@@ -1,31 +1,115 @@
-# Anticrm Platform
-
-[![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+# Anticrm Platform [![Maintainability](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/maintainability)](https://codeclimate.com/github/hardcoreeng/platform/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/5cb6d2d426619568816b/test_coverage)](https://codeclimate.com/github/hardcoreeng/platform/test_coverage)
+
+Business Applications Reimagined. Real time. Web first. Open source.
+
+
+**Description:** 
+
+The PLATFORM is an attempt to reimagine a concept of web platforms for business needs like WordPress, Bitrix, etc.
```

Comment:
I do not think we need this paragraph at this particular moment.

Reply:
fixed